wikibase/1.30/bundle/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM ubuntu as fetcher
+
+RUN apt-get update && \
+    apt-get install --yes --no-install-recommends unzip
+
+ADD https://extdist.wmflabs.org/dist/extensions/OAuth-REL1_30-352260e.tar.gz .
```

Comment:
These URLs are not permanent and will change on you. Specifically, if someone backports a change to REL1_30, the sha1 will change, so the URL does too. You can use <https://www.mediawiki.org/w/api.php?action=query&list=extdistbranches&edbexts=OAuth&formatversion=2&format=json> to get the correct URL for REL1_30.

Reply:
Awesome, thanks; that's really helpful to know!